Source/Utils/WarpXUtil.cpp

```diff
@@ -392,6 +398,39 @@ parseStringtoInt(std::string str, std::string name)
     return ival;
 }
 
+int
+parseSignalNameToNumber(const std::string &str)
+{
+    IParser signals_parser(str);
+
+    for (int i = 0; i < 32; ++i) {
+#if defined(__GNU_LIBRARY__)
+        // Returns upper-case, no "SIG"
+        std::string name_upper = sigabbrev_np(i);
```

Comment:
Grrr, this function was only introduced recently, in GNU libc 2.32. In Ubuntu, that's only available from 21.04 and later, so no LTS release yet. Fedora 34 and up has it.

Reply:
Might need to just enumerate the relevant signals ourselves, or import a bit of public domain code:
https://github.com/util-linux/util-linux/blob/master/lib/signames.c